tracing/ops_tracing/vendor/charms/tempo_coordinator_k8s/v0/tracing.py

```diff
@@ -950,15 +950,17 @@ def charm_tracing_config(
      proceed with charm tracing (with or without tls, as appropriate)
 
     Usage:
-    >>> from lib.charms.tempo_coordinator_k8s.v0.charm_tracing import trace_charm
-    >>> from lib.charms.tempo_coordinator_k8s.v0.tracing import charm_tracing_config
-    >>> @trace_charm(tracing_endpoint="my_endpoint", cert_path="cert_path")
-    >>> class MyCharm(...):
-    >>>     _cert_path = "/path/to/cert/on/charm/container.crt"
-    >>>     def __init__(self, ...):
-    >>>         self.tracing = TracingEndpointRequirer(...)
-    >>>         self.my_endpoint, self.cert_path = charm_tracing_config(
-    ...             self.tracing, self._cert_path)
+
+        from lib.charms.tempo_coordinator_k8s.v0.charm_tracing import trace_charm
+        from lib.charms.tempo_coordinator_k8s.v0.tracing import charm_tracing_config
+        @trace_charm(tracing_endpoint="my_endpoint", cert_path="cert_path")
+        class MyCharm(...):
+            _cert_path = "/path/to/cert/on/charm/container.crt"
+            def __init__(self, ...):
+                self.tracing = TracingEndpointRequirer(...)
+                self.my_endpoint, self.cert_path = charm_tracing_config(
+                    self.tracing, self._cert_path
+                )
```

Comment:
Technically, I should provide a clean for this upstream, but we're considering reimplementing this library rather than vendoring it, and there's the "move all the interface libraries" project as well, so it doesn't seem worth doing.